WalletWasabi.Fluent/Behaviors/ScrollToSelectedItemBehavior.cs

```diff
@@ -12,7 +13,30 @@ protected override void OnAttachedToVisualTree(CompositeDisposable disposable)
 		if (AssociatedObject is { SelectionInteraction: { } selection, RowSelection: { } rowSelection })
 		{
 			Observable.FromEventPattern(selection, nameof(selection.SelectionChanged))
-				.Select(x => rowSelection.SelectedIndex.FirstOrDefault())
+				.Select(x =>
+				{
+					var selectedIndexPath = rowSelection.SelectedIndex.FirstOrDefault();
+					if (AssociatedObject.Rows is null)
+					{
+						return selectedIndexPath;
+					}
```

Comment:
I am wondering if `Rows` is null, how any selection is even possible? Shouldn't the checking for null should be in L13 with the others?

Reply:
No, as rows can change after the behavior was attached.